RecoPixelVertexing/PixelVertexFinding/src/PixelVertexProducer.cc

```diff
@@ -41,7 +41,7 @@ PixelVertexProducer::PixelVertexProducer(const edm::ParameterSet& conf)
     track_pt_min   = PVcomparerPSet.getParameter<double>("track_pt_min");    
     if (track_pt_min != ptMin_) {
       if (track_pt_min < ptMin_)
-	edm::LogWarning("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
+	edm::LogInfo("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
```

Comment:
This warning was inserted by @mtosi with the PV comparer (https://github.com/davidlange6/cmssw/commit/d2543fec4254dfe91006a262663d14265a4a707b).
I agree to silent it because I think it is not a problem if the PV comparer and PixelVertexProducer use different pT min threshold. 
In the current menu it gives the warning message because track_pt_min [1] and the PtMin [2] are different in the online b-tag sequence, but it was expected [3]. 
[1] http://cmslxr.fnal.gov/source/HLTrigger/Configuration/python/HLT_GRun_cff.py#0303
[2] http://cmslxr.fnal.gov/source/HLTrigger/Configuration/python/HLT_GRun_cff.py#7814
[3] https://its.cern.ch/jira/browse/CMSHLT-128